runelite-api/src/main/java/net/runelite/api/Varbits.java

```diff
@@ -563,7 +563,12 @@
 	TWISTED_LEAGUE_RELIC_2(10050),
 	TWISTED_LEAGUE_RELIC_3(10051),
 	TWISTED_LEAGUE_RELIC_4(10052),
-	TWISTED_LEAGUE_RELIC_5(10053);
+	TWISTED_LEAGUE_RELIC_5(10053),
+
+	/**
+	 * PVP
```

Comment:
This should explain the expected function and values of this varbit, not just be a section header of sorts. (Not all other entries in this file are a good reference; I would expect this to look more like `GE_OFFER_CREATION_TYPE` on L542 than the sections above)